zerver/finders.py

```diff
@@ -11,7 +11,7 @@ class ExcludeUnminifiedMixin(object):
     def list(self, ignore_patterns):
         # We can't use ignore_patterns because the patterns are
         # applied to just the file part, not the entire path
-        excluded = '^(js|styles|templates)/'
+        excluded = '^(js|templates)/'
```

Comment:
When I checked out old pipeline's documentation, they didn't mention adding anything to STATICFILES_FINDERS, but the new pipeline's documentation tells us to add `pipeline.finders.PipelineFinder` to it.

Maybe this means that the old pipeline finds out things by itself, so it has no problems. But the new pipeline tries to use `ZulipStorage` and is unable to find files in the 'styles' folder.